src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -274,6 +274,14 @@ public static Task<RazorLanguageServer> CreateAsync(Stream input, Stream output,
             return Task.FromResult(razorLanguageServer);
         }
 
+        // Today we only extract services from the language server in scenarios we know are eventually going away.
+        // For instance, at the time of writing this comment we're extracting some endpoitns from the server for
+        // perf concerns to avoid doing StreamJsonRpc hops to resolve things like an applicable language at a location.
+        // Eventually language queries will be done only at the language server layer at which point inproc/vs out of proc
```

Comment:
```suggestion
        // Eventually language queries will be done only at the language server layer at which point inproc versus out of proc
```

Just to avoid ambiguity with Visual Studio :)